docs/hazmat/primitives/asymmetric/dh.rst

```diff
@@ -143,6 +143,9 @@ Key interfaces
 
     .. versionadded:: 0.9
 
+    DH private key objects have methods from this interface as well as
+    :class:`DHPrivateKeyWithSerialization`.
```

Comment:
Do you mean the ordering? As in "DH private key objects have methods from DHPrivateKeyWithSerialization in addition to those on this interface" rather than the current phrasing?